e2e/tests/e2e_happy_path/HappyPath.spec.ts

```diff
@@ -188,11 +193,18 @@ suite('Display source code changes in the running application', async () => {
 
     test('Build application with changes', async () => {
         await runTask('che: build');
-        await projectTree.expandPathAndOpenFileInAssociatedWorkspace(projectName, 'build.txt');
+        await projectTree.expandPathAndOpenFileInAssociatedWorkspace(projectName, 'result-build.txt', 300000);
+        await editor.waitText('result-build.txt', '[INFO] BUILD SUCCESS');
+        
+        //workaround for issue: https://github.com/eclipse/che/issues/14771
+
+        /*await projectTree.expandPathAndOpenFileInAssociatedWorkspace(projectName, 'build.txt');
```

Comment:
The same question https://github.com/eclipse/che/pull/14814#pullrequestreview-299997228